QuickReference.ipynb

```diff
@@ -0,0 +1,402 @@
+    "\n",
+    "The position (`pos`) is best specified with `SkyCoord` objects (see http://docs.astropy.org/en/stable/api/astropy.coordinates.SkyCoord.html). \n",
+    "\n",
+    "The size of the region is specified with the `radius` keyword and may be decimal degrees or an Astropy `Angle` (http://docs.astropy.org/en/stable/api/astropy.coordinates.Angle.html#astropy.coordinates.Angle). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m51_pos = SkyCoord.from_name(\"m51\")\n",
+    "\n",
+    "services = vo.regsearch(servicetype='conesearch', keywords='usno-b')\n",
+    "results = services[0].search(pos=m51_pos, radius=0.1)\n",
+    "results.to_table()  # Astropy Table is often useful for handling and displaying cone search results."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2.3 Image search\n",
+    "Example:  Find an image search service for GALEX, and search it around coordinates 13:37:00.950,-29:51:55.51 (M83) with a radius of .2 degrees.  Download the first file in the results.\n",
+    "#### Find an image service"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "services = vo.regsearch(servicetype='image', keywords=['galex'])\n",
+    "services.to_table()['ivoid', 'short_name', 'res_title']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Search one of the services\n",
+    "The first service looks good.  Search it!\n",
+    "\n",
+    "For more details on using `SkyCoord` see http://docs.astropy.org/en/stable/api/astropy.coordinates.SkyCoord.html#astropy.coordinates.SkyCoord\n",
+    "\n",
+    "**NOTE**:  For image searches, the size of the region is defined by the `size` keyword which is more like a diameter than a radius."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m83_pos = SkyCoord('13h37m00.950s -29d51m55.51s')\n",
+    "results = services[0].search(pos=m83_pos, size=.2)\n",
+    "\n",
+    "# We can look at the results.\n",
+    "results.to_table()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Download an image\n",
+    "For the first result, print the file format and download the file. If repeatedly executing this code, add `cache=True` to `download_file()` to prevent repeated downloads.\n",
+    "\n",
+    "See `download_file()` documentation here: https://docs.astropy.org/en/stable/api/astropy.utils.data.download_file.html#astropy.utils.data.download_file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(results[0].format)\n",
+    "file_name = download_file(results[0].getdataurl())  \n",
```

Comment:
Nothing to be done about it, but we should be aware that this took a bit to execute for me on my home (i.e. slow) internet, so it will probably be a slow cell during the workshop.

Reply:
If time allows, I can look for a way to seamlessly choose a smaller file, though probably not a jpeg.